### DISCUSSION TITLE
Displaying data within a JSON column

### DISCUSSION THREAD

**Discussion by sawoods** [Budibase/budibase#5897] [Help]
From the docs (  https://docs.budibase.com/docs/json ) I'm expecting to see columns for the keys in the the JSON field but its not working for me.

Am I missing something? Anybody else experiencing this issue?

I've attached the database file

![screenshot_24](https://user-images.githubusercontent.com/13916656/168476555-dbf44ee0-4824-4ce7-a0c0-905f741ac083.png)

![screenshot_26](https://user-images.githubusercontent.com/13916656/168476735-4deca013-4964-4fbf-99d0-bd664d249ed1.png)

![screenshot_25](https://user-images.githubusercontent.com/13916656/168476645-36ce0bbc-0e16-48cf-8a4d-173ed3c9b337.png)

[JSONTest-export-1652622552611.txt](https://github.com/Budibase/budibase/files/8695084/JSONTest-export-1652622552611.txt)

**Comment by melohagan** [comment]
Hi @sawoods 

Thank you very much for the screenshots and the app export. 

This is a bug, which I have raised here: https://github.com/Budibase/budibase/issues/5911
